src/components/calltree/ProfileCallTreeContextMenu.js

```diff
@@ -68,13 +83,26 @@ class ProfileCallTreeContextMenu extends PureComponent {
       case 'copyFunctionName':
         this.copyFunctionName();
         break;
+      case 'copyUrl':
+        this.copyUrl();
+        break;
       case 'copyStack':
         this.copyStack();
         break;
     }
   }
 
   render() {
+    const isJS = () => {
```

Comment:
I'd prefer to avoid an allocation in render if possible, could you switch this to a small function or just inline the logic?